packages/fxa-shared/tracing/node-tracing.ts

```diff
@@ -0,0 +1,152 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import {
+  TraceExporter as GcpTraceExporter,
+  TraceExporterOptions as GcpTraceExporterOptions,
+} from '@google-cloud/opentelemetry-cloud-trace-exporter';
+import { getNodeAutoInstrumentations } from '@opentelemetry/auto-instrumentations-node';
+import { ExportResult } from '@opentelemetry/core';
+import { JaegerExporter } from '@opentelemetry/exporter-jaeger';
+import { registerInstrumentations } from '@opentelemetry/instrumentation';
+import { Resource } from '@opentelemetry/resources';
+import {
+  BatchSpanProcessor,
+  ConsoleSpanExporter,
+  ReadableSpan,
+  SimpleSpanProcessor,
+} from '@opentelemetry/sdk-trace-base';
+import { NodeTracerProvider } from '@opentelemetry/sdk-trace-node';
+import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions';
+
+import { ILogger } from '../log';
+import { TracingOpts } from './config';
+
+const log_type = 'node-tracing';
+
+/** Overloaded GcpTraceExporter that removes PII from traces. */
+export class FxaGcpTraceExporter extends GcpTraceExporter {
+  constructor(config?: GcpTraceExporterOptions) {
+    super(config);
+  }
+
+  override export(
+    spans: ReadableSpan[],
+    resultCallback: (result: ExportResult) => void
+  ) {
+    for (const span of spans) {
+      // Don't record raw db statemets. They often leak PII.
+      if (span.attributes['db.statement']) {
+        span.attributes['db.statement'] = '[FILTERED]';
```

Comment:
This is just an example of how we can scrub data. It's probably too simple. Ideally, we would scrub just the values that look like PII. It's TBD to what extent we do this and how conservative we will be.